test/support/scenario_support.rb

```diff
@@ -72,9 +72,7 @@ def record(name, options = {})
   end
 
   def destroy_repo_if_exists(repo)
-    if exists? { Katello.pulp_server.resources.repository.retrieve(repo.pulp_id) }
-      Katello.pulp_server.resources.repository.delete(repo.pulp_id)
-    end
+    # TODO: Implement for Pulp 3
```

Comment:
Don't worry about re-implementing this, it can be taken out.  We have all the support methods we need in `test/support/pulp3_support.rb`. The only method that uses it is `create_repo` in the same file. I believe all other uses of `create_repo` in the tests actually refer to the `create_repo` in the Pulp 3 test support class. Too bad it has the same name!

Reply:
Not sure, It is referenced in https://github.com/Katello/katello/blob/a9e44e26abb667ed474fc562b97172a62ea98a2a/test/scenarios/scenario_test.rb#L18-L39 but that does have a `skip`. If it was completely unreferenced I'd have removed it.

What's your preference here?

Reply:
We can remove the test, it looks to me like everything tested in there is tested elsewhere.